apps/nextjs-website/src/editorialComponents/Footer/components/ProductType.tsx

```diff
@@ -0,0 +1,14 @@
+import * as t from 'io-ts';
```

Comment:
I think you should install `io-ts` as dependency if you want to use it. I'd avoid to use the `io-ts` of another package (in this case `mui-italia`)